includes/class-query.php

```diff
@@ -245,10 +269,8 @@ public function is_activitypub_request() {
 			return $this->is_activitypub_request;
 		}
 
-		global $wp_query;
-
 		// One can trigger an ActivityPub request by adding ?activitypub to the URL.
-		if ( isset( $wp_query->query_vars['activitypub'] ) ) {
+		if ( $this->has_query_var() ) {
```

Comment:
Could we check for `$_GET['activitypub']` instead?

Reply:
Oh my god 🫣